generators/app/index.js

```diff
@@ -4,139 +4,177 @@
  * Drizzle Project Yeoman Generator
  *
  * @example yo drizzle
+ * @example yo drizzle --full
+ * @example yo drizzle --help
  */
 
 const chalk = require('chalk');
 const inquirer = require('inquirer');
 const yeoman = require('yeoman-generator');
-const yosay = require('yosay');
+const utils = require('./utils');
 
-const separator = label =>
-  new inquirer.Separator(`--- ${label} ---`);
+const INTRO = `
+${chalk.bold.underline('Welcome to your new Drizzle project.')}
```

Comment:
I get wary when I see too many JS expressions inside of template strings, but I don't have a better idea here.